src/module/applications/api/pseudo-document-sheet.mjs

```diff
@@ -100,7 +101,7 @@ export default class PseudoDocumentSheet extends HandlebarsApplicationMixin(Appl
 
   /**
    * The pseudo-document. This can be null if a parent pseudo-document is removed.
-   * @type {PseudoDocument|null}
+   * @type {TPsuedo|null}
```

Comment:
```suggestion
   * @type {TPseudo|null}
```

Reply:
I don't think this is true anymore actually, since implementing the auto-close of pseudo-document sheets.